src/metabase/lib/aggregation.cljc

```diff
@@ -317,17 +317,11 @@
     :operator (:short aggregation-operator)
     :args [column]}))
 
-(def ^:private SelectedColumnMetadata
```

Comment:
Since this is used in multiple places now and since it was optional I just rolled `:selected?` into `ColumnMetadata` itself